clarity/src/vm/costs/mod.rs

```diff
@@ -896,6 +896,19 @@ impl LimitedCostTracker {
             Self::Free => ExecutionCost::max_value(),
         }
     }
+    // Set the limit on the Limited tracker, or do nothing if its free.
+    // Returns the previous limit if it was set.
+    pub fn set_limit(&mut self, limit: ExecutionCost) -> Option<ExecutionCost> {
+        match self {
+            Self::Limited(ref mut data) => {
+                let old_limit = data.limit.clone();
+                data.limit = limit;
+                Some(old_limit)
+            }
+            Self::Free => None,
+        }
+    }
```

Comment:
It seems like this method isn't used anywhere -- can it be dropped from the PR?

Reply:
Yes. Was leftover from a prior iteration :)